### ISSUE TITLE
[FEATURE] TPSSync Module (git-a28b236)

### ISSUE THREAD

**Issue by mumy-255** [CCBlueX/LiquidBounce#2428]
### LiquidBounce Branch

Nextgen

### Describe your feature request.

In some servers, due to TPS being below 20 but modules still executing at their original frequency, this can lead to problematic issues (such as difficulty in landing critical hits with KillAura due to hit cooldowns, and being unable to set values higher than 1). 

It is recommended to add a TPSSync module, which calculates the server's TPS (for example, by calculating the frequency of WorldTimeUpdateS2CPacket, or by capturing information from the Tab list or scoreboard, or using the /tps command), and then adjusts the module's working frequency based on the TPS (e.g., adjusting cooldowns and HurtTime for KillAura). 

However, different servers may not necessarily use the same method, so it is suggested to have configurable modes.

**Comment by SenkJu** [comment]
I don't think this can be done in a way that would actually be beneficial in reality.

**Comment by mumy-255** [comment]
> I don't think this can be done in a way that would actually be beneficial in reality.

Perhaps you could add a Custom mode to allow users to customize the simulated TPS rate.
(At least this is much better than repeatedly toggling KillAura to ensure critical hits.)
